presentation/presentation.go

```diff
@@ -890,3 +897,12 @@ func (p *Presentation) addCustomRelationships() {
 	p.ContentTypes.AddOverride("/docProps/custom.xml", "application/vnd.openxmlformats-officedocument.custom-properties+xml")
 	p.Rels.AddRelationship("docProps/custom.xml", unioffice.CustomPropertiesType)
 }
+
+// Close closes the presentation, removing any temporary files that might have been
+// created when opening a document.
+func (p *Presentation) Close() error {
+	if p.TmpPath != "" {
```

Comment:
would be good to have the check that it starts with os.TempDir() also?  I think we had that in some places
strings.HasPrefix(p.TmpPath, os.TempDir()) or something like that?  remember seeing that

Reply:
os.TempDir is relevant only for disk storage and I moved this check here https://github.com/unidoc/unioffice/pull/423/files#diff-9158c05ebf44c9e9d0aa98532dfa5c4d to the RemoveAll method